specs/StartServers.groovy

```diff
@@ -561,6 +563,9 @@ class StartServers extends PluginTestHelper {
         then:
         assert runProcedureJob.getStatus() == 'error'
         assert runProcedureJob.getUpperStepSummary() =~ "Failed to connect to the controller"
+        
+        cleanup:
```

Comment:
can it be easier to just provide incorrect controller ip and have the same errors?

Reply:
as for me this scenario is more realistic